__DEFINES/callback.dm

```diff
@@ -0,0 +1,4 @@
+// Set a callback's thing_to_call to GLOBAL_PROC in order to call a global proc.
+#define GLOBAL_PROC "fuck you duny"
+
+#define CALLBACK new /datum/callback
```

Comment:
I'm not sure if hiding the fact that you're creating a datum is the right thing to do.

Reply:
ok, let's not

Reply:
Changed the path to /callback so it's still short, removed the macro